manifests/c/CyberCat/ADBAppControl/1.8.3/CyberCat.ADBAppControl.locale.en-US.yaml

```
@@ -1,12 +1,12 @@
-# Created with Komac v1.9.1
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
 
 PackageIdentifier: CyberCat.ADBAppControl
-PackageVersion: 1.8.2
+PackageVersion: 1.8.3
 PackageLocale: en-US
 Publisher: Cyber.Cat
 PackageName: ADB AppControl
 License: ©2020-2021 ADB AppControl by Cyber.Cat

```

```suggestion
License: Proprietary
```
A copyright isn't a license